app/views/foods/index.html.erb

```diff
@@ -0,0 +1,17 @@
+Searched item count: <%= @total_food_count %>
+<% @foods_info[0..9].each do |food_info| %>
```

Comment:
This `@foods_inof[0..9]` is breaking the SRP of the view and it shows our code is inefficient. Essentially, we only want to ever make 10 food objects, so this should happen in the facade like so:
```ruby
searched_food_info[:foods].first(10).map do |food_info|
        Food.new(food_info)
      end
```
This way we aren't creating unused objects.